components/schemas/stacks/spec/StackSpec.yml

```diff
@@ -24,6 +24,11 @@ properties:
     type: array
     items:
       $ref: StackSpecTestContainer.yml
+  scoped_variables:
+    nullable: true
+    type: array
+    items:
+      $ref: ../../scoped-variables/ScopedVariable.yml
```

Comment:
This is very incorrect, and will not work. The scoped variable you're referencing here is an implemented, concrete one requiring an ID and a bunch more. The stack spec is not that, it's a blueprint and therefore the variables defined here are only partly the same. This is why we have stack spec separated out entirely. 

![image](https://github.com/cycleplatform/api-spec/assets/5110855/468b62de-a388-4a27-af02-803b452facf2)

Please redo this and create stack spec specific variables.

Reply:
<img width="848" alt="image" src="https://github.com/cycleplatform/api-spec/assets/112718756/2b7c0554-455d-4ac3-8376-65ac68392e72">